src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/AnsiConsole.cs

```diff
@@ -9,34 +9,19 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer, bool useConsoleColor)
+        private AnsiConsole(TextWriter writer)
         {
             Writer = writer;
-
-            _useConsoleColor = useConsoleColor;
-            if (_useConsoleColor)
-            {
-                OriginalForegroundColor = Console.ForegroundColor;
-            }
         }
 
         private int _boldRecursion;
-        private bool _useConsoleColor;
 
-        public static AnsiConsole GetOutput(bool useConsoleColor)
-        {
-            return new AnsiConsole(Console.Out, useConsoleColor);
-        }
+        public static AnsiConsole Output = new AnsiConsole(Console.Out);
```

Comment:
The original code had a method that created the instance each time it was called. It was changed to a public static field. It should be a public static property with just a getter that creates a new console each time or, if we care, create `private static readonly` (or lazily initialized) fields that would be returned from static properties.